docs/semgrep-app/findings.md

```diff
@@ -133,18 +133,31 @@ Display and filter for findings reported only in specific time. To display findi
 
 ### Grouping by rule
 
-Semgrep Cloud Platform enables you to group findings by a specific rule on the Findings page.
+Semgrep Cloud Platform by default groups findings by a specific rule on the Findings page. Procedure documented in this section helps you to enable group by rule when it was disabled.
```

Comment:
Since it's on by default, how about it just ends on:

```suggestion
Semgrep Cloud Platform by default groups findings by a specific rule on the Findings page. 
```

Remove the procedure to group findings by rules and just indicate that they can change back, because it's pointless to have 2 separate procedures just pressing a toggle.